test/wormhole.test.ts

```diff
@@ -271,12 +259,70 @@ describe('Wormhole', () => {
   })
 
   describe('flush', () => {
-    it('pays back debt')
-    it("can't flush not-configured domain")
+    it('pays back debt (negative debt)', async () => {
+      // Burn L2 DAI (without withdrawing DAI on L1)
+      await l2WormholeBridge.connect(l2User).initiateWormhole(mainnetDomain, userAddress, amt, userAddress)
+      expect(await l2WormholeBridge.batchedDaiToFlush(mainnetDomain)).to.be.eq(amt)
+      expect(await mainnetSdk.dai.balanceOf(l1Escrow.address)).to.be.eq(amt)
+
+      // Pay back (not yet incurred) debt. Usually relaying this message would take 7 days
+      await relayMessagesToL1(l2WormholeBridge.connect(l2User).flush(mainnetDomain))
+
+      expect(await l2WormholeBridge.batchedDaiToFlush(mainnetDomain)).to.be.eq(0)
+      expect(await mainnetSdk.dai.balanceOf(l1Escrow.address)).to.be.eq(0)
+      expect(await mainnetSdk.dai.balanceOf(router.address)).to.be.eq(0)
+      expect(await mainnetSdk.dai.balanceOf(join.address)).to.be.eq(0)
```

Comment:
can we add here (and in the next testcase) an assert against debt ceiling here? dssWormholeJoin.debt and internal accounting in vat (`dai`?)